### ISSUE TITLE
Server Error on Time Travel

### ISSUE THREAD

**Issue by KelleyDrake** [ddrake/Integrated_Farm_Budget_Tool#139]
I was playing around with my Demo budget this morning and went back in time to 1/10/2023 to set up a published budget.   When I set the date to 1/10/23  I got a server 500 error.   I then rolled forward the locked date to try 2/10/2023 instead and that date worked fine.    Just thought I'd pass that along.  Not sure what might be causing that server 500 error for the January date and not the February date.

**Comment by ddrake** [comment]
Thanks— I’ll check it out!

Sent from my iPad

On Sep 22, 2023, at 7:35 AM, KelleyDrake ***@***.***> wrote:

﻿

I was playing around with my Demo budget this morning and went back in time to 1/10/2023 to set up a published budget. When I set the date to 1/10/23 I got a server 500 error. I then rolled forward the locked date to try 2/10/2023 instead and that date worked fine. Just thought I'd pass that along. Not sure what might be causing that server 500 error for the January date and not the February date.
